compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/interop/UIKitInteropContainer.uikit.kt

```diff
@@ -38,35 +40,124 @@ internal val LocalUIKitInteropContainer = staticCompositionLocalOf<UIKitInteropC
     error("UIKitInteropContainer not provided")
 }
 
+
+internal enum class UIKitInteropState {
```

Comment:
IMHO, `interop` and enum values are too generic.
Could you please add brief comment explaining why this enum used?

Reply:
Yeah, sure